src/Compilers/CSharp/Portable/Symbols/MissingCorLibrarySymbol.cs

```diff
@@ -79,5 +79,20 @@ internal override bool RuntimeSupportsNumericIntPtr
                 throw ExceptionUtilities.Unreachable;
             }
         }
+
+        internal override bool RuntimeSupportsByRefFields
+        {
+            get
+            {
+                // For now we assume that it is not supported by default
```

Comment:
For now? Is this ever going to change?

Reply:
Just copied previous property ;-)
To answer your question, this whole property will be removed once TargetingFramework.Net70 gets added, so we likely won't change the default.